Resources/Locale/en-US/devices/device-network.ftl

```diff
@@ -24,12 +24,20 @@ device-frequency-prototype-name-surveillance-camera-entertainment = Entertainmen
 device-address-prefix-vent = VNT-
 device-address-prefix-scrubber = SCR-
 device-address-prefix-sensor = SNS-
+# Damn bet you couldn't see this one coming.
+device-address-prefix-teg = TEG-
+device-address-prefix-heater = HTR-
+device-address-prefix-freezer = FZR-
+device-address-prefix-volume-pump = VPP-
```

Comment:
why not gas pumps too

Reply:
Do I look like I bothered to map those on my test map

Reply:
even outside of teg its still sick